bedrock/base/templates/macros.html

```diff
@@ -642,3 +642,84 @@
   </div>{#-- /.container --#}
 </header>{#-- /#fxfamilynav-header --#}
 {%- endmacro %}
+
+{% macro send_to_device(platform, include_title=True, title_text='', include_logo=False) %}
+  {% set ios_link = '#' %}
+  {% set android_link = settings.GOOGLE_PLAY_FIREFOX_LINK %}
+  <section id="send-to-device" class="{% if include_logo %}logo {% endif %}{% if include_title %}title{% else %}no-title{% endif %}">
+    <div class="form-container">
+      {% if include_title %}
+        {% if title_text %}
+          <h2 class="form-heading">{{ title_text }}</h2>
+        {% else %}
+          <h2 class="form-heading">{{ _('Send Firefox to your smartphone or tablet') }}</h2>
+        {% endif %}
+      {% endif %}
+      <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
+      <form id="send-to-device-form" action="{{ secure_url('firefox.send-to-device-post') }}" method="post"{% if platform == 'select' %} class="dropdown"{% endif %}>
+        <ul class="error-list hidden">
+          <li class="sms">{{ _('Sorry. This number isn’t valid. Please enter a U.S. phone number.') }}</li>
+          <li class="email">{{ _('Please enter an email address.') }}</li>
+          <li class="platform">{{ _('Please make a selection before proceeding.') }}</li>
+          <li class="system">{{ _('An error occurred in our system. Please try again later.') }}</li>
+        </ul>
+        <div class="input">
+          <div class="platform-container">
+            {% if platform == 'select' %}
+            {# User selects OS they would like to receive link for. #}
+              <label for="id-platform">
+                {{ _('Select iOS or Android') }}
+              </label>
+              <select id="id-platform" name="platform" autocomplete="off" required>
+                <option value="" selected>{{ _('Choose OS') }}</option>
+                <option value="ios">iOS</option>
+                <option value="android">Android</option>
+              </select>
+            {% elif platform == 'ios' %}
+            {# User receives link for iOS. #}
+              <input type="hidden" id="id-platform" name="platform" value="ios">
+            {% elif platform == 'android' %}
+              {# User receives link for Android. #}
+              <input type="hidden" id="id-platform" name="platform" value="android">
+            {% else %}
+              {# User receives links for both iOS and Android. #}
+              <input type="hidden" id="id-platform" name="platform" value="all">
```

Comment:
Could maybe set `platform` to `all` if no value is explicitly passed in, then have a single hidden input like:

`<input type="hidden" id="id-platform" name="platform" value="{{ platform }}">`